src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,56 @@
+.stopwatch {
+  &__clockface {
+    width: 500px;
+    height: 500px;
+    border: 5px solid black;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__center {
+    width: 30px;
+    height: 30px;
+    background-color: black;
+    border-radius: 50%;
+    position: relative;
+  }
+
+  &__seconds {
+    width: 249px;
+    height: 10px;
+    background-color: green;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin-top: -5px;
```

Comment:
Consider using 'transform: translateY(-50%);' instead of 'margin-top: -5px;' for better centering.